src/syscalls.proto

```diff
@@ -56,7 +56,7 @@ message GithubRest {
 }
 
 message GithubRestResponse {
-  string data = 1;
+  bytes data = 1;
```

Comment:
But if it's a tarball, for example, it _won't_ be a string, it'll be a byte array